cmd/delete.go

```diff
@@ -1,54 +1,47 @@
 package cmd
 
 import (
-	"errors"
-	"os"
-
+	"bytes"
 	"fmt"
+	"html/template"
 
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
-// deleteCmd represents the create command
+// deleteCmd represents the delete command
 var deleteCmd = &cobra.Command{
-	Use:   "delete",
-	Short: "Delete a namespace",
-	Long:  `This command delete a namespace and all the associated resources`,
+	Use:   "delete [command]",
+	Short: "Delete an object",
+	Long: `Delete resources by namespace or names.
 
+Deletetion of a namespace will remove the namespace all his attached object and the intentory attached to it. While the remove of the resource will only supress it form the namespace.`,
```

Comment:
Deletetion of a namespace will delete the namespace and remove all his attached object with the intentory attached to it. While removing an object will only supress it from the namespace but keep everything else.`,